actions/server.search.created.at.relative.yaml

```diff
@@ -0,0 +1,87 @@
+---
+description: Query servers
+enabled: true
+entry_point: src/server_query_actions.py
+name: server.search.created.at.relative
+parameters:
+  timeout:
+    default: 5400
+  submodule:
+    default: search_servers_older_than_relative_to_now
+    enum:
+      - "search_servers_older_than_relative_to_now"
+      - "search_servers_younger_than_relative_to_now"
```

Comment:
Does this make sense to have? I can't imagine how this works without a time machine :sweat_smile:

Reply:
Instead I think this should take a ISO8601 style time (without the TZ stuff), or default to `now` as a paremter then it makes sense?

This might be feature creep so feel free to put it into the TODO pile

Reply:
> Does this make sense to have? I can't imagine how this works without a time machine sweat_smile

The `search_servers_younger_than_relative_to_now` means we search for servers younger than a timestamp that is calculated from `days` `hours` `minutes` `seconds` in the past from the current timestamp
so it is necessary - it doesn't find servers younger than current timestamp - which I guess you interpreted it ass

Reply:
I'm going to change this function in a future PR - leaving it for now